tests/reconfiguration.py

```diff
@@ -242,6 +242,16 @@ def get_nodes(status):
     return network
 
 
+@reqs.description("Get node CCF version")
+def test_version(network, args):
+    nodes = network.get_joined_nodes()
+
+    for node in nodes:
+        with node.client() as c:
+            r = c.get("/node/version")
+            assert r.body.json()["version"] == args.ccf_version
```

Comment:
The naming feels a little vague here, the "node" is effectively CCF+app, and this only talks about the CCF version, so I think either the path or the name should mention "ccf" or "framework" somewhere.